src/labels/CustomLabels.labels

```diff
@@ -7655,6 +7663,102 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Gift Processing configuration error</shortDescription>
         <value>Your org is missing required configuration for Gift Processing. Contact your admin for help.</value>
     </labels>
+    <labels>
+        <fullName>gsApplicationStatus</fullName>
+        <categories>Getting Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Application Status label for the application status component</shortDescription>
+        <value>Application Status: </value>
+    </labels>
+    <labels>
+        <fullName>gsApplicationStatusModalHeader</fullName>
+        <categories>Getting Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Header of the modal label for the application status component</shortDescription>
+        <value>Check Application Status</value>
+    </labels>
+    <labels>
+        <fullName>gsApplyForFreeLicenses</fullName>
+        <categories>Getting Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Apply for licenses button label for the application status component</shortDescription>
+        <value>Apply for Salesforce Licenses</value>
+    </labels>
+    <labels>
+        <fullName>gsCheckStatus</fullName>
+        <categories>Getting Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Check Status label for the application status component</shortDescription>
+        <value>Check Status</value>
+    </labels>
+    <labels>
+        <fullName>gsClose</fullName>
```

Comment:
Why have two labels for a button label for close? There was one above with an api name of "Close" that I requested to be changed to "btnClose". Why not just have one label for "btnClose" that is used **everywhere** for a close button?